index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+   <head>
+<script src="script.js"></script>       
+
+   <body>
```

Comment:
You should properly indent children elements. The script element should be intended from the head. You also shouldn't have that blank line there.